sound/soc/intel/boards/sof_nau8825.c

```diff
@@ -190,11 +190,6 @@ static int sof_card_late_probe(struct snd_soc_card *card)
 	struct sof_hdmi_pcm *pcm;
 	int err;
 
-	if (list_empty(&ctx->hdmi_pcm_list))
```

Comment:
@yongzhi1 you need to split this out into a seprate patch. The fixes tag won't apply otherwise.

Reply:
> @yongzhi1 you need to split this out into a seprate patch. The fixes tag won't apply otherwise.

For the 2nd patch, I do not need to put fixes to 8d08726f(ASoC: Intel: add sof-nau8825 machine driver) do I?

Reply:
no, that's fine.